Symfony/src/AppBundle/Resources/views/Dashboard/index.html.twig

```diff
@@ -0,0 +1,58 @@
+{% extends 'base.html.twig' %}
+
+{% block stylesheets %}
+    <!-- Custom CSS -->
+    {% include 'AppBundle:Dashboard:css.html.twig' %}
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <h3>Welcome, {{ user.username }}.
+            {% if tier != null %}
+                You are currently a {{ tier.tierName }} user.
```

Comment:
Let's make that `Your current tier is ...`.

For us it's indeed a user, but from their perspective they are a person not a "user" so the message should be informative for the user, not us.